### DISCUSSION TITLE
Multiple tables with Nav Bar

### DISCUSSION THREAD

**Discussion by ChiangPin** [l-lin/angular-datatables#1522] [Q&A] [answered]
Hi, is that possible to integrate bootstrap Nav Tab to the table?  Like this, so user can navigate to the table they wish to.

PS: I did position absolute to the Show Entries and Search, it didnt worked..

![image](https://user-images.githubusercontent.com/37894622/110597070-110f1280-81bb-11eb-959a-dce8dcd5ee7a.png)

**Comment by shanmukhateja** [comment, marked as answer]
Hi,

I don't think DataTables allows for adding external HTML nodes to the table.

Instead, I would recommend moving the tabs to top of the table element as "parent" nodes and render tabular content acc. to selected tab selected.

You could also use multiple table instances for each individual Nav Tab and use `ngIf` to selectively show the appropriate table.
